src/plugins/discover/public/application/angular/doc_table/infinite_scroll.ts

```diff
@@ -34,11 +34,12 @@ export function createInfiniteScrollDirective() {
         const isMobileView = document.getElementsByClassName('dscSidebar__mobile').length > 0;
         const usedScrollDiv = isMobileView ? scrollDivMobile : scrollDiv;
         const scrollTop = usedScrollDiv.scrollTop();
+        const scrollOffset = usedScrollDiv.prop('offsetTop') || 0;
 
         const winHeight = Number(usedScrollDiv.height());
         const winBottom = Number(winHeight) + Number(scrollTop);
         const elTop = $element.get(0).offsetTop || 0;
-        const remaining = elTop - winBottom;
```

Comment:
when the browser window height was limited the remaining calculation was always larger then the scroll div height, so the next 50 docs where never displayed